ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/components/MyJobsFilter.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Switch } from '@patternfly/react-core';
+
+export type MyJobsFilterProps = {
+    showOnlyMyJobs: boolean;
+    onToggle: (checked: boolean) => void;
+};
+
+function MyJobsFilter({ showOnlyMyJobs, onToggle }: MyJobsFilterProps) {
+    return (
+        <Switch
+            id="view-only-my-jobs"
+            label="View only my jobs"
+            labelOff="View only my jobs"
```

Comment:
Can you double-check the text when the switch is off?
PatternFly example seems to imply that the text described the opposite state, maybe View all jobs?
https://www.patternfly.org/components/switch#basic

Reply:
Aha, so a different implementation of the same idea here:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/MainPage/Header/OrchestratorComponentsToggle.tsx#L26-L34

A comment above `return` statement will help our future selves to understand.

Reply:
Yea, a comment might make it clearer. This is what I'm thinking:

`
We're using the same label for both "label" and "labelOff" because changing the label between "on" and "off" states was causing confusion. When the label changes (e.g., from "View only my jobs" to "View all jobs"), users found it unclear what state the switch was in and what they were actually viewing. By keeping the label consistent, it avoids this confusion and maintains clarity on what the switch controls. 
`